app/client/jsx/IFrameRoom.jsx

```diff
@@ -2,10 +2,22 @@ import React from 'react'
 import JitsiVideo from './JitsiVideo.jsx'
 
 export default props => {
-    const { src } = props.iframeOptions
+    /* secondSrc is TEMPORARY TST-SPECIFIC THING */
+    const { src, secondSrc } = props.iframeOptions
+    const left = secondSrc ? (
+        <div className="iframe-section">
+                <iframe
+                    src={secondSrc}
+                    height="100%"
+                    width="100%"
+                    frameBorder="0px"
+                >
+                </iframe>
+            </div>
+    ) : <JitsiVideo jitsiData={props.jitsiData}></JitsiVideo>
     return (
         <div className="iframe-room">
-            <JitsiVideo jitsiData={props.jitsiData}></JitsiVideo>
+            {left}
```

Comment:
This was so that I could put two iframes in one room. This is a hack for now, but I want to be able to extend the config to take multiple iframes -- or even better, change the way we do rooms so that they're arbitrarily composable with iframes, video, chat, content, etc.